main/res/drawable/zoomout.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item android:drawable="@drawable/white_bcg"/>
+    <item
```

Comment:
here as well...

Reply:
As the ripple is applied to the background XML, it will work for both zoom buttons.